frontend/src/components/entity/EntityCard.tsx

```diff
@@ -114,6 +115,30 @@ const EntityCard = ({
               compact={compact}
               config={entityTypeConfig}
             />
+            {!isAvailable && compact && (
+              <Box
+                display="flex"
+                justifyContent="center"
+                alignItems="center"
+                position="absolute"
+                top="0"
+                color="white"
+                bgcolor="rgba(0,0,0,.6)"
+                width="100%"
+                height="100%"
+                sx={{
+                  [theme.breakpoints.down('sm')]: {
+                    fontSize: '0.6rem',
+                  },
+                }}
+              >
+                <Typography textAlign="center" fontSize="inherit">
+                  {entity.type == TypeEnum.VIDEO
+                    ? t('video.notAvailableAnymore')
+                    : t('entityCard.thisElementIsNotAvailable')}
+                </Typography>
+              </Box>
```

Comment:
The same components are also used by `<EntitySelectorInnerAuth>`.

Let's factorize them in their own component.